serverless/openvino/omz/public/mask_rcnn_inception_resnet_v2_atrous_coco/nuclio/function.yaml

```diff
@@ -2,7 +2,7 @@
 # have enough memory (more than 4GB). Look here how to do that
 # https://stackoverflow.com/questions/44417159/docker-process-killed-with-cryptic-killed-message
 metadata:
-  name: openvino.omz.public.mask_rcnn_inception_resnet_v2_atrous_coco
+  name: openvino-mask-rcnn-inception-resnet-v2-atrous-coco
```

Comment:
could you keep `omz-public` in the name?

Reply:
Error - Service "nuclio-openvino-omz-public-mask-rcnn-inception-resnet-v2-atrous-coco" is invalid: metadata.name: Invalid value: "nuclio-openvino-omz-public-mask-rcnn-inception-resnet-v2-atrous-coco": must be no more than 63 characters